packages/orion/src/Tour/index.js

```diff
@@ -0,0 +1,260 @@
+import React, { useState, useEffect } from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import Reactour from 'reactour'
+import { disableBodyScroll, enableBodyScroll } from 'body-scroll-lock'
+
+import { Button } from '..'
+
+import Anchor from './Anchor'
+import Badge from './Badge'
+import Portal from './Portal'
+import TourModal from './TourModal'
+import {
+  getAnchorClassName,
+  BadgeDistance,
+  BadgePosition,
+  useBadgePosition
+} from './utils'
+
+const DEFAULT_PADDING = 0
+const DEFAULT_RADIUS = 4
+
+function parseSteps(steps) {
+  return steps.map(
+    (
+      {
+        selector,
+        anchor,
+        title,
+        content,
+        padding,
+        badgePosition,
+        badgeDistance,
+        position,
+        radius,
+        actionBefore,
+        actionAfter
+      },
+      i
+    ) => {
+      const parsedContent = (
+        <>
+          {title && <h1 className="orion-tour-title">{title}</h1>}
+          {content}
+        </>
+      )
+      if (anchor) {
+        return {
+          selector: `.${getAnchorClassName(i)}`,
+          content: parsedContent,
+          anchor,
+          position,
+          actionBefore,
+          actionAfter
+        }
+      }
+
+      return {
+        selector,
+        content: parsedContent,
+        padding,
+        badgePosition,
+        badgeDistance,
+        position: position || 'right',
+        radius,
+        actionBefore,
+        actionAfter
+      }
+    }
+  )
+}
+
+function Tour({
+  className,
+  steps,
+  welcomeModal,
+  dismissButtonContent,
+  nextButtonContent,
+  prevButtonContent,
+  finishButtonContent,
+  onFinish,
+  onDismiss
+}) {
+  const [tourSteps, setTourSteps] = useState(() => parseSteps(steps))
+  const [currentStep, setCurrentStep] = useState(0)
+  const [openTour, setOpenTour] = useState(!welcomeModal)
+  const [openModal, setOpenModal] = useState(!!welcomeModal)
+  const badgePosition = useBadgePosition(tourSteps, currentStep)
+
+  useEffect(() => {
+    disableBodyScroll()
+    return () => enableBodyScroll()
+  }, [])
+
+  useEffect(() => {
+    setTourSteps(parseSteps(steps))
+  }, [steps, setTourSteps])
+
+  function nextStep() {
```

Comment:
Como é uma função, é sempre interessante começar com um verbo.
```suggestion
  function handleNextStep() {
```

Reply:
Fui influenciado pelo nome da prop do `reactour`. Acho tua sugestão melhor.
- [ ] `handleNextStep`